src/interface/obsidian/src/modal.ts

```diff
@@ -90,11 +90,42 @@ export class KhojModal extends SuggestModal<SearchResult> {
     }
 
     async renderSuggestion(result: SearchResult, el: HTMLElement) {
+        // Max Size (words or number of lines)
+        let lines_to_render = 8;
         let words_to_render = 30;
-        let entry_words = result.entry.split(' ')
-        let entry_snipped_indicator = entry_words.length > words_to_render ? ' **...**' : '';
-        let snipped_entry = entry_words.slice(0, words_to_render).join(' ');
-        MarkdownRenderer.renderMarkdown(snipped_entry + entry_snipped_indicator, el, null, null);
+        var path = "";
+        var basename = "";
+
+        // set path and basename of file depending on OS
+        if (result.file.indexOf('\\') > -1) {
+            path = result.file.split('\\').slice(0, -1).join('\\');
+            basename = result.file.split('\\').slice(-1)[0];
+        }
+        if (result.file.indexOf('/') > -1) {
+            path = result.file.split('/').slice(0, -1).join('/');
+            basename = result.file.split('/').slice(-1)[0];
+        }
+        basename = basename.replace('.md', '');
```

Comment:
1. We should leave the files `.md` extension in, to act as an indicator that the string is a filename. It'll also be a useful indicator later if/when we support searching for other file types (e.g for images in the users obsidian vault)
2. We don't need to extract `path` based on my other comment
3. Based on above two points, this code can be simplified like below:
```suggestion
        // Extract filename of result
        let os_path_separator = result.file.includes('\\') ? '\\' : '/';
        let filename = result.file.split(os_path_separator).pop();
```